muppet.js

```diff
@@ -97,6 +103,15 @@ function configure() {
     if (cfg.logLevel)
         log.level(cfg.logLevel);
 
+    if (opts.metricsPort) {
+        if (isNaN(opts.metricsPort)) {
```

Comment:
This check should be more stringent: the port must be an integer between 1024 and 49151.

Reply:
Thank you for taking a look. The check condition has updated as requested.